apps/storybook2/src/stories/molecules/AuthWrapper.stories.tsx

```
@@ -0,0 +1,79 @@
+            </Label>
+          </div>
+          <div className="mb-7">
+            <Button size="2" type="submit" className="w-full p-2 sm:w-auto">
+              Create account
+            </Button>
+          </div>
+          <p className="font-medium text-gray-500 text-sm dark:text-gray-400">
+            Already have an account?&nbsp;
+            <a
+              href="#"
+              className="text-primary-700 hover:underline dark:text-primary-500"
```

Biome (1.9.4)

[error] Provide a valid value for the attribute href.

The href attribute should be a valid a URL
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)